src/components/profile/style/profile.less

```diff
@@ -23,6 +23,10 @@
     margin: 0;
   }
 
+  .ga-profile-icon {
+    font-family: FontAwesome;
```

Comment:
the icon for elevation differences needs to have different size from the icons of elev up and down.
do I put here font-size and use this class only for icons elev up and down?